ui/src/vega.tsx

```diff
@@ -66,10 +66,13 @@ export const
       init = () => {
         const el = ref.current
         if (!el) return
+
+        const spec = JSON.parse(state.specification)
+        if (!isNaN(spec.height)) el.style.height = `${spec.height + 10}px`
```

Comment:
Any reason why the extra 10px is necessary here? Would be good to have a comment here explaining why.

Reply:
Also, would be good to include `{}` for the conditionals.

Reply:
Thanks, good point, always forget about the comments somehow.

> Also, would be good to include {} for the conditionals

I thought the current style is inline ifs if possible without brackets.